src/libraries/Microsoft.PowerFx.Core/Entities/External/TableDelegationInfo.cs

```diff
@@ -299,6 +302,29 @@ public enum SummarizeMethod
         CountRows
     }
 
+    [Obsolete("preview")]
+    public class CountCapabilities
+    {
+        public CountCapabilities()
+        {
+        }
+
+        public virtual bool IsCountableTable()
+        {
+            return false;
+        }
+
+        public virtual bool IsCountableAfterJoin()
```

Comment:
can you add a comment for more detail. 

Pattern here is good - assume not delegable and we need to opt-in to allow it. 

Fyi, see [https://learn.microsoft.com/en-us/power-platform/power-fx/reference/function-table-counts#syntax](https://learn.microsoft.com/en-us/power-platform/power-fx/reference/function-table-counts#syntax) 

CountRows(table) and CountIf(table, ..)  are different operations and may have different support

Reply:
good catch! Thanks!